src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -100,8 +194,7 @@ struct CharacteristicEvolution {
       const Parallel::CProxy_ConstGlobalCache<Metavariables>&
           global_cache) noexcept {
     auto& local_cache = *(global_cache.ckLocalBranch());
-    if (next_phase == Metavariables::Phase::RegisterWithObserver or
-        next_phase == Metavariables::Phase::Evolve) {
+    if (next_phase == Metavariables::Phase::Evolve) {
```

Comment:
Why do you have this `if` statement, can't you just remove it? Phases that don't have an action list will just be skipped anyway

Reply:
that is not true, `execute_next_phase` is called in every phase except the initialization phase.
If all that is done is calling `start_phase`, that just broadcasts to the component which will then change to the next phase and execute its action list for that phase, which in the case would do nothing (except the broadcast) so removing the if is not wrong...